app/init.php

```diff
@@ -767,6 +767,25 @@ function (mixed $value) {
     return $group;
 });
 
+$register->set('db', function () {
+    // This is usually for our workers or CLI commands scope
+       $dbHost = App::getEnv('_APP_DB_HOST', '');
+       $dbPort = App::getEnv('_APP_DB_PORT', '');
+       $dbUser = App::getEnv('_APP_DB_USER', '');
+       $dbPass = App::getEnv('_APP_DB_PASS', '');
+       $dbScheme = App::getEnv('_APP_DB_SCHEMA', '');
+
+       $pdo = new PDO("mysql:host={$dbHost};port={$dbPort};dbname={$dbScheme};charset=utf8mb4", $dbUser, $dbPass, array(
+           PDO::ATTR_TIMEOUT => 3, // Seconds
+           PDO::ATTR_PERSISTENT => true,
+           PDO::ATTR_DEFAULT_FETCH_MODE => PDO::FETCH_ASSOC,
+           PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION,
+           PDO::ATTR_EMULATE_PREPARES => true,
+           PDO::ATTR_STRINGIFY_FETCHES => true,
+       ));
+
+       return $pdo;
+});
```

Comment:
why do we need this? you can use the db connection from the database pool directly

Reply:
We are missing setting up PDO in migrate file. This is required for using PDO during migrations

Reply:
We can remove this now we are using `updateAttribute`, we should try keep moving towards using only the db lib functions :+1:

Reply:
keeping it here @abnegate  for migration files that require PDO